server/sonar-server/src/main/java/org/sonar/server/ws/CacheWriter.java

```diff
@@ -23,17 +23,23 @@
 import java.io.StringWriter;
 import java.io.Writer;
 import org.apache.commons.io.IOUtils;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
 
 /**
  * Writer that writes only when closing the resource
  */
 class CacheWriter extends Writer {
+  private static final Logger LOGGER = Loggers.get(CacheWriter.class);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "LOGGER" private field. ([squid:S1068](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=fix%2Fteryk%2Fws-stream-log-error&issues=AWDgPLB_bBlXo5JPxg7l&open=AWDgPLB_bBlXo5JPxg7l)

[ISSUE_KEY]: AWDgPLB_bBlXo5JPxg7l

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED